### ISSUE TITLE
Call for aiopg maintainer

### ISSUE THREAD

**Issue by asvetlov** [aio-libs/aiopg#384]
The project needs more love.
Unfortunately I am and @jettify are overwhelmed by other Open Source projects.
I personally sped most spare time on aiohttp issues.

We need a responsible person for monitoring  aiopg issues and reviewing pull requests.

I don't want to give up the project support.

The plan is:
* a volunteer's duty is answering on issues and taking careful review.
* all non-trivial changes approved by this guy also require approval from me or @jettify for trial period at least.
* I'm and @jettify are BDFLs for the project

Anybody want to bear the duty?

**Comment by 1st1** [comment]
I'd at least consider deprecating the project in favour of asyncpg.

**Comment by DeoLeung** [comment]
`asyncpg` looks fancy but for OLAP application there seems not a strong benefit to move from `aiopg` considering the engineering cost.

**Comment by jettify** [comment]
I do not think we are going to deprecate this project, `asyncpg` looks nice and stuff, but
1) It is different form usual DBAPI spec
2) there are existing projects that relies on it
3) psycopg2 is almost standard postgres driver for python

**Comment by tvoinarovskyi** [comment]
@jettify What should we do with aiomysq, are you still supporting it? Any alternative out there?
I ask cause those are quite similar projects, at least from the API perspective.

**Comment by elprans** [comment]
@jettify FWIW it should be possible to implement a DBAPI-like facade on top of `asyncpg` that is fully compatible with `aiopg`.  The main issue here is the duplication of effort in the asyncio-specific parts and things like the connection pool.

**Comment by jettify** [comment]
@elprans Agreed it would be nice to have `asyncpg` as backend.

**Comment by vir-mir** [comment]
@asvetlov I can try, i also use in work, you can write off in the mail virmir49 gmail

**Comment by jettify** [comment]
@asvetlov we need more help here so if you do not mind I will add @vir-mir to the the committers. He knows aiopg internals and contributed large PR https://github.com/aio-libs/aiopg/pull/219 it was not merged mostly because lack of time on our part.

**Comment by asvetlov** [comment]
Agree

**Comment by vir-mir** [comment]
@jettify what further actions? I do not yet have access.

**Comment by jettify** [comment]
@vir-mir sent you invitation let me know if you have access (you should be able to merge https://github.com/aio-libs/aiopg/pull/219 ).

Please follow this simple rules:

1) on each change please create pull request
2) each PR should be reviewed and tested
3) if PR is not commented for more than 3-5 days by other maintainers, feel free to merge it
4) be nice and respectful to others :)

https://github.com/aio-libs/aiohttp/blob/master/CONTRIBUTING.rst#how-to-become-an-aiohttp-committer

And wellcome to aio-libs team!

**Comment by vir-mir** [comment]
@jettify Thank you! access is!

**Comment by JohnTitor** [comment]
@asvetlov @jettify I'm not an expert, but interested in aiopg. I'd like to join.

**Comment by webknjaz** [comment]
@JohnTitor you could try going through issues and see what you can do

**Comment by JohnTitor** [comment]
@webknjaz OK, I'll try. Thank you.
